.prettierignore

```diff
@@ -7,7 +7,6 @@ public
 
 *.lock
 *.log
-*.test.ts
```

Comment:
<img width="1135" alt="Screenshot 2023-08-21 at 6 38 42 PM" src="https://github.com/calcom/cal.com/assets/1780212/9de6400b-3eae-486a-898c-a61a5aa91421">
Was setting these lint errors in test files. So, I am removing test files from `.prettierignore`. If we discover any issues due to that, we should fix the root cause. cc @leog

Reply:
Had to revert the .prettierignore change because it would require fixing prettier errors in many test files. Job for a different PR.